$NetBSD: patch-HUnit.cabal,v 1.1 2011/07/28 09:54:12 phonohawk Exp $

Don't build/install the test executables.
It's no use for any library users.

--- HUnit.cabal.orig	2010-09-27 07:52:00.000000000 +0000
+++ HUnit.cabal
@@ -51,44 +51,3 @@ Library
         Test.HUnit.Text,
         Test.HUnit
     Extensions: CPP
-
-Executable basic-tests
-    Main-Is:            HUnitTest98.lhs
-    HS-Source-Dirs:     . tests
-    Build-Depends:      base<5
-    if flag(base4)
-        Build-Depends: base >=4
-        CPP-Options: -DBASE4
-        GHC-Options: -Wall
-    else
-        Build-Depends: base <4
-    if impl(ghc >= 6.10)
-        Build-Depends: base >=4
-        Extensions: CPP
-
-Executable extended-tests
-    Main-Is:            HUnitTestExtended.lhs
-    HS-Source-Dirs:     . tests
-    Build-Depends:      base<5
-    if flag(base4)
-        Build-Depends: base >=4
-        CPP-Options: -DBASE4
-        GHC-Options: -Wall
-    else
-        Build-Depends: base <4
-    if impl(ghc >= 6.10)
-        Build-Depends: base >=4
-        Extensions: CPP
-
-Executable terminal-tests
-    Main-Is:            TerminalTest.lhs
-    HS-Source-Dirs:     . tests
-    Build-Depends:      base<5
-    if flag(base4)
-        Build-Depends: base >=4
-        CPP-Options: -DBASE4
-    else
-        Build-Depends: base <4
-    if impl(ghc >= 6.10)
-        Build-Depends: base >=4
-        Extensions: CPP
